roles/pulp_devel/templates/cli.settings.toml.j2

```diff
@@ -0,0 +1,8 @@
+[cli]
+base_url = "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_fqdn }}"
+username = "admin"
+password = "{{ pulp_default_admin_password }}"
+cert = "{{ pulp_webserver_disable_https | default(false) | ternary('', '{{ pulp_certs_dir }}/pulp_webserver.crt') }}"
+key = "{{ pulp_webserver_disable_https | default(false) | ternary('', '{{ pulp_certs_dir }}/pulp_webserver.key') }}"
```

Comment:
I don't think, those are the right certs to use as a connecting client.
Maybe just drop those lines.

Reply:
what are the right ones?

Reply:
They are only there for client certificates (as used by katello)